launch

```diff
@@ -81,6 +81,8 @@ CONFIG = {
                 'binary': 'qemu-mimiker-mipsel',
                 'options': [
                     '-device', 'VGA',
+                    # '-drive', 'if=none,id=stick,file=usb_drive.img',
```

Comment:
Why is it commented out?
If you mean to leave it here, add one more line of comment about why is that